### PR TITLE
feat(styles): icon tabs header horizon delta theming

### PR DESCRIPTION
**Pull request by g-cheishvili**
## Related Issue
part of #3213

## Description
Icon tabs bar update

## Breaking Changes

* Overflow button instead of positioning itself using `position: absolute`, now positions using `margin-left: auto`. It now is always in the end of container

### CODE REVIEW

**Review by platon-rov — COMMENTED** [review]
* Container padding is different

Current:
image

Specs:
image

* Label color is grey what is incorrect.

Current:
image

Specs:
image

* Icons not centered, for not selected tab glyph color is wrong

Current:
image

Specs:
image

File: src/styles/theming/sap_horizon.scss [hunk 1]
```diff
@@ -322,6 +322,10 @@
   --fdIcon_Tab_Bar_Semantic_Badge_Color_Negative: var(--sapNegativeElementColor);
   --fdIcon_Tab_Bar_Semantic_Badge_Color_Critical: var(--sapCriticalElementColor);
   --fdIcon_Tab_Bar_Semantic_Badge_Color_Informative: var(--sapInformativeElementColor);
+  --fdIcon_Tab_Bar_Overflowing_Padding: 0.188rem;
```
[the remark this answers is not in the document]
Comment (g-cheishvili) [reply src/styles/theming/sap_horizon.scss]: Padding of the header is set with sizing. By default fd-icon-tab-bar does not have any padding

[the remark this answers is not in the document]
Comment (g-cheishvili) [reply src/styles/theming/sap_horizon.scss]: For some reason Label color and font is correctly set, but visually it does not appear. Although in theming variables they exist

**Review by platon-rov — DISMISSED** [review]
> Padding of the header is set with sizing. By default fd-icon-tab-bar does not have any padding

I see, but accordingly to the specifications size S should be applied for small screens, M for medium and so on, and we don't apply it implicitly. It's weirdo to don't have paddings at all and i don't see anything about this variant without paddings in the wiki, maybe we should contact designers, what do you think?

> For some reason Label color and font is correctly set, but visually it does not appear. Although in theming variables they exist

Inactive tab should have `color: --sapTextColor`, but it has `color: var(--sapContent_LabelColor)`.

image

**Review by platon-rov — COMMENTED** [review]
Accidentally approved.

**Comment by g-cheishvili** [comment]
> > Padding of the header is set with sizing. By default fd-icon-tab-bar does not have any padding
> 
> I see, but accordingly to the specifications size S should be applied for small screens, M for medium and so on, and we don't apply it implicitly. It's weirdo to don't have paddings at all and i don't see anything about this variant without paddings in the wiki, maybe we should contact designers, what do you think?
> 
> > For some reason Label color and font is correctly set, but visually it does not appear. Although in theming variables they exist
> 
> Inactive tab should have `color: --sapTextColor`, but it has `color: var(--sapContent_LabelColor)`.
> 
> image

Theme changer glitches, if you change theme to horizon, then refresh page, it will load correct variables and then you'll see that although correct vars are set, it still shows differently from wiki

**Review by InnaAtanasova — APPROVED** [review]

**Pull request merged**